src/views/Lottery/components/YourPrizesCard/PrizesWonContent.tsx

```diff
@@ -52,12 +56,13 @@ const PrizesWonContent: React.FC = () => {
       const txHash = await onMultiClaim()
       // user rejected tx or didn't go thru
       if (txHash) {
+        afterClaim()
```

Comment:
For success/fail handlers lets use a standard naming scheme. `onSuccess` and `handleSuccess`

Reply:
Done